code/jasmin/mlkem_ref/gen_matrix.jazz

```diff
@@ -11,6 +12,7 @@ export fn gen_matrix_jazz(reg u64 ap, reg u64 seedp, reg u64 transposed)
 
   for i = 0 to MLKEM_SYMBYTES
   {
+    #[declassify]
```

Comment:
This is needed because the function is exported, but does the function need to be exported?

Reply:
Precisely. This is a test function. It needs its `seedp` argument to point to public data. The only way to specify this is (currently) to add a declassification annotation on the memory loads.